### PR TITLE
Feature/pda 100

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/TaskDetails/TaskDetailsContainer.js [hunk 1]
```diff
@@ -67,23 +67,26 @@ class TaskDetailsContainer extends React.Component {
...
   async fetchTask() {
-    const { config } = this.state;
+    const { config, loadingTask } = this.state;
     const { taskId } = this.props;
 
     const connection = (config && config.knowledgeSource) || '';
     const [, containerId] = (config && config.process && config.process.split('@')) || '';
     const taskContainerId = `${taskId}@${containerId}`;
 
+    if (!loadingTask) this.setState({ loadingTask: true });
```
Comment (rimselis) [inline widgets/src/components/TaskDetails/TaskDetailsContainer.js]: strange that this did not trigger eslint, let's use `{ }`

File: widgets/src/components/TaskList/TaskListContainer.js [hunk 2]
```diff
@@ -223,15 +225,19 @@ TaskList.propTypes = {
...
   frameId: PropTypes.string,
 };
 
 TaskList.defaultProps = {
   classes: {},
-  lazyLoading: false,
+  lazyLoading: true,
   onError: () => {},
+  onSelectTask: row => {
+    console.log('test', row);
```
Comment (rimselis) [inline widgets/src/components/TaskList/TaskListContainer.js]: still needed?

Comment (gReis89) [reply widgets/src/components/TaskList/TaskListContainer.js]: good catch

**Comment by entando-service** [comment]
All is well

**Review by rimselis — APPROVED** [review]

**Pull request merged**